tensorflow_addons/metrics/f_scores.py

```diff
@@ -184,10 +185,8 @@ def get_config(self):
         return {**base_config, **config}
 
     def reset_states(self):
-        self.true_positives.assign(tf.zeros(self.init_shape, self.dtype))
-        self.false_positives.assign(tf.zeros(self.init_shape, self.dtype))
-        self.false_negatives.assign(tf.zeros(self.init_shape, self.dtype))
-        self.weights_intermediate.assign(tf.zeros(self.init_shape, self.dtype))
+        reset_value = tf.zeros(self.init_shape, dtype=self.dtype)
+        K.batch_set_value([(v, reset_value) for v in self.variables])
```

Comment:
```python
tf.keras.backend.batch_set_value([(v, reset_value) for v in self.variables])
```

Reply:
Just for the consistent style in the repo :-)

Reply:
Yes probably it is more popular/consistent but not uniform. :wink: 
See https://github.com/tensorflow/addons/search?q=%22backend+as+K%22&unscoped_q=%22backend+as+K%22

Reply:
Thanks!

Looking at the other codes in tesnorflow_addons, its seem to be used inconsistently. 

If you choose one style, I will modify it to be consistent. :)